README.md

```diff
@@ -38,6 +38,21 @@ This library uses [`umzug`'s CLI](https://github.com/sequelize/umzug#cli-usage)
 ```sh
 npx migrate-contentful -h
 ```
+
+## Customising migration paths
+
+The default migrations folder is `migrations/scripts`
+
+When creating a migration, you can use the `--folder` flag to specify a custom folder to look for migrations in.
+```
+npx migrate-contentful --folder <path>
```

Comment:
Did you mean `npx migrate-contentful create --folder <path>`?

Reply:
Yes I did! 

Thanks for catching this 🙂